dotnet3.5/S3/UploadObjectExample/UploadObject/UploadObject.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. 
+// SPDX-License-Identifier: MIT-0
+
+// snippet-start:[s3.dotNET.UploadObject]
```

Comment:
Is this snippet tag being actively used by anyone? If not, I wouldn't include it.

Reply:
Removed the snippet.